controllers/urls.js

```diff
@@ -5,18 +5,30 @@
 const Url = require('../models/Url')
 const User = require('../models/User')
 
-exports.index = async (req, res) => {
-  try {
-    const links = await Url.find()
-    const users = await User.find()
-    res.render('urls', {
-      title: 'URLs',
-      urlInfo:links,
-      currentUser:req.user.id,
-      userInfo:users,
-      visits: links.clickCounter,
-    })
-  } catch(error) {
-    console.error(error)
+module.exports = {
+  index: async (req, res) => {
+    try {
+      const links = await Url.find()
+      const users = await User.find()
+      res.render('urls', {
+        title: 'URLs',
+        urlInfo:links,
+        currentUser:req.user.id,
+        userInfo:users,
+        visits: links.clickCounter,
+      })
+    } catch(error) {
+      console.error(error)
+    }
+  },
+  deleteUrl: async (req, res) => {
+    console.log(req.body.slugFromJSFile)
+    try {
+        await Url .findOneAndDelete({slug:req.body.slugFromJSFile})
```

Comment:
This whitespace between Url and .findOneAndDelete() method in unintended. Will re-push